lib/assets/stylesheets/chef/components/logo-font.scss

```diff
@@ -0,0 +1,25 @@
+@font-face {
+  font-family: 'chef';
+  src: url('../fonts/chef.eot?-8vpdz');
+  src: url('../fonts/chef.eot?#iefix-8vpdz') format("embedded-opentype"), url('../fonts/chef.woff?-8vpdz') format("woff"), url('../fonts/chef.ttf?-8vpdz') format("truetype"), url('../fonts/chef.svg?-8vpdz#chef') format("svg");
+  font-weight: normal;
+  font-style: normal;
+}
+
+
+.icon {
```

Comment:
I think this should be `.icon-chef`, since it's very possible that somebody could write code that has an `icon` class in it and unexpected things would happen.

Also, could you please add some doc examples at the top so it can be found on https://style.chef.io? You just need comments similar to what is in https://github.com/chef/chef-web-core/blob/master/lib/assets/stylesheets/chef/components/icon.scss